sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -232,28 +231,39 @@ private void performLogging(LogLevel logLevel, LogLevel environmentLogLevel, boo
      * Determines if the environment and logger support logging at the given log level.
      *
      * @param logLevel Logging level for the log message.
-     * @param environmentLoggingLevel Logging level the environment is set to support.
      * @return Flag indicating if the environment and logger are configured to support logging at the given log level.
      */
-    private boolean canLogAtLevel(LogLevel logLevel, LogLevel environmentLoggingLevel) {
+    public boolean canLogAtLevel(LogLevel logLevel) {
```

Comment:
`LogLevel` is in an implementation package and will need to be moved to public package.

Reply:
The PR is for reviewing the code flow, not finished yet.
Will merge the changes in PR:  https://github.com/Azure/azure-sdk-for-java/pull/6886.
Also, will fix the tests after merging.